grype/db/v6/affected_package_store.go

```diff
@@ -411,15 +403,11 @@ func (s *affectedPackageStore) resolveDistro(d OSSpecifier) ([]OperatingSystem,
 	query := s.db.Model(&OperatingSystem{})
 
 	if d.Name != "" {
-		query = query.Where("name = ?", d.Name)
-	}
-
-	if d.Codename != "" {
-		query = query.Where("codename = ?", d.Codename)
+		query = query.Where("name = ? OR release_id = ?", d.Name, d.Name)
```

Comment:
Is `release_id` always going to be queried by `d.Name`? Is there every a chance `release_id` could be the value that's in `d.LabelVersion`?

I guess codename and names never really overlap in the distro space - I just wasn't sure if this case existed at all.

Reply:
label version represents either a codename or something like "unstable" (a non numeric version). The release ID doesn't stand in for the version, this states more of a distribution family and shouldn't change from release to release like the version will.